2_adventures/001/alexander-ruehle/03_code_kata_karate_chop/chop_test.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'chop'
+require 'test/unit'
+
+class ChopTest < Test::Unit::TestCase
```

Comment:
'test/unit' is now 'Minitest'

And the inheritance is now 'Minitest::Test`

Using require 'minitest/autorun' does a bit for you, allowing mixed style tests (spec and unit style)